src/App.js

```diff
@@ -76,8 +82,8 @@ class App extends Component {
   }
 
   connectClients(config) {
-    this.mqtt.connect(config.mqttBrokerUrl);
-    this.json.setServerUrl(config.domoticzUrl);
+    this.mqtt.connect(config.mqttBrokerUrl, config.mqttLogin, config.mqttPassword);
```

Comment:
This doesn't do anything unless you also modify the MqttClientSingleton. Maybe you forgot to add that bit to the pull request ?